code/game/machinery/slotmachine.dm

```diff
@@ -26,6 +28,8 @@
 	var/working = 0
 	var/balance = 0 //How much money is in the machine, ready to be CONSUMED.
 	var/jackpots = 0
+	var/paymode = HOLOCHIP //It is a config
+	var/cointype = /obj/item/coin/iron //default cointype
```

Comment:
but why

Reply:
The cointype? Just so I dont have to mess around with undeclared vars in the rest of the code.
If it is about the holochip config, is it so you can still use coins if you are inclined to do so